configs/polyfills/brorand.js

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
converted from cjs-like to esm from https://github.com/indutny/brorand (needed for rollup bundling)